fedocal/__init__.py

```diff
@@ -1030,7 +1038,9 @@ def view_meeting_page(meeting_id, full):
         tzone=tzone,
         title=meeting.meeting_name,
         editor=editor,
-        from_date=from_date)
+        from_date=from_date,
+        reminder_at_options=APP.config.get('ICAL_REMINDER_AT_OPTIONS') or []
```

Comment:
@pypingou I know, but 
``reminder_at_options=APP.config.get('ICAL_REMINDER_AT_OPTIONS') or []``
handles the worst case scenario when
```
config[ICAL_REMINDER_AT_OPTIONS'] = None
```

It's unnecessary here. It's a side effect of writing fault tolerant code for E Commerce products to handle stupidity from some other sources ;). Here, it's unnecessary, I believe. I will fix it.